rita/src/rita_client/traffic_watcher/mod.rs

```diff
@@ -101,16 +92,52 @@ pub fn watch<T: Read + Write>(
     trace!("exit price {}", exit_price);
 
     if destinations.contains_key(&exit.mesh_ip) {
+        let client = reqwest::Client::builder()
+            .timeout(Duration::from_secs(5))
+            .build()?;
+
+        let client_tx = SystemTime::now();
+        let RTTimestamps { exit_rx, exit_tx } = client
+            .get(&format!(
+                "http://[{}]:{}/rtt",
+                exit.mesh_ip,
+                match SETTING.get_exit_client().get_current_exit() {
+                    Some(current_exit) => current_exit.registration_port,
+                    None => {
+                        error!("No current exit even though an exit route is present");
+                        return Err(format_err!(
+                            "No current exit even though an exit route is present"
```

Comment:
couldn't we not duplicate that string somehow? Minor nit.

Reply:
if we print out the return value from this function, the error! would be unnecessary